liquibase-couchbase/src/test/java/liquibase/ext/couchbase/validator/JsonChangelogValidatorTest.java

```diff
@@ -0,0 +1,188 @@
+package liquibase.ext.couchbase.validator;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.networknt.schema.ErrorMessageType;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.ValidationMessage;
+import com.networknt.schema.ValidatorTypeCode;
+import liquibase.Scope;
+import liquibase.ext.couchbase.exception.InvalidJSONException;
+import liquibase.ext.couchbase.exception.ResourceNotFoundException;
+import liquibase.resource.Resource;
+import liquibase.resource.ResourceAccessor;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+import java.io.InputStream;
+import java.util.HashSet;
+import java.util.Set;
+
+import static java.util.Collections.emptySet;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class JsonChangelogValidatorTest {
+
+    @Mock
+    private ObjectMapper objectMapper;
+    @Mock
+    private JsonSchemaFactory schemaFactory;
+    @InjectMocks
+    private JsonChangelogValidator validator;
+
+    @Test
+    @SneakyThrows
+    void Should_validate_json_by_schema_successfully() {
+        String pathToChangelog = "pathToChangelog";
+        String pathToSchema = "pathToSchema";
+
+        Scope scope = mock(Scope.class);
+        ResourceAccessor resourceAccessor = mock(ResourceAccessor.class);
```

Comment:
You use similar list of mocked classes in every method. Why not to move them to object level as private fields?

Reply:
++ Thanks. Done